src/components/FeaturePanel/PublicTransport/LineNumber.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+interface LineNumberProps {
+  name: string;
+  color: string;
+}
+
+/**
+ * A function to determine whether the text color should be black or white
+ * @param hexBgColor The background color in hex format, e.g. #ff0000
+ * @returns 'black' or 'white' depending on the brightness of the background color
+ */
+function whiteOrBlackText(hexBgColor) {
+  const r = parseInt(hexBgColor.slice(1, 3), 16);
+  const g = parseInt(hexBgColor.slice(3, 5), 16);
+  const b = parseInt(hexBgColor.slice(5, 7), 16);
+  const brightness = (r * 299 + g * 587 + b * 114) / 1000;
+  return brightness > 125 ? 'black' : 'white';
+}
+
+export const LineNumber: React.FC<LineNumberProps> = ({ name, color }) => {
+  let bgcolor: string;
+  if (!color) bgcolor = '#898989';
```

Comment:
This color works good for dark mode, could please place eg. `#ddd` for light mode?

This is too dark in light mode:
![image](https://github.com/zbycz/osmapp/assets/385047/49223180-e2d6-43b1-8988-09ee86f3f9a9)